https://dawn-review.googlesource.com/c/dawn/+/126120

From a7423b3d8367a706c50ab787df2fe5eedb6c3c9f Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Mon, 03 Apr 2023 13:51:27 +0000
Subject: [PATCH] IWYU: add cstdint for uint8_t dawn::native::stream::ByteVectorSink

Bug: chromium:957519
Change-Id: I48436db0e203d793e47d717bfb75977c67145e94
Reviewed-on: https://dawn-review.googlesource.com/c/dawn/+/126120
Reviewed-by: Corentin Wallez <cwallez@chromium.org>
Commit-Queue: Stephan Hartmann <stha09@googlemail.com>
Kokoro: Kokoro <noreply+kokoro@google.com>
--- a/third_party/dawn/src/dawn/native/stream/ByteVectorSink.h
+++ b/third_party/dawn/src/dawn/native/stream/ByteVectorSink.h
@@ -15,6 +15,7 @@
 #ifndef SRC_DAWN_NATIVE_STREAM_BYTEVECTORSINK_H_
 #define SRC_DAWN_NATIVE_STREAM_BYTEVECTORSINK_H_
 
+#include <cstdint>
 #include <ostream>
 #include <vector>
 
--- chromium-108.0.5359.125/third_party/dawn/src/tint/reader/spirv/namer.h.orig	2022-12-14 02:29:58.489729000 +0100
+++ chromium-108.0.5359.125/third_party/dawn/src/tint/reader/spirv/namer.h	2023-05-10 15:01:33.296517619 +0200
@@ -18,6 +18,7 @@
 #include <string>
 #include <unordered_map>
 #include <vector>
+#include <cstdint>
 
 #include "src/tint/reader/spirv/fail_stream.h"
 
